zulip_botserver/zulip_botserver/server.py

```diff
@@ -65,7 +74,8 @@ def handle_bot(bot):
     message_handler.handle_message(message=event["message"],
                                    bot_handler=restricted_client,
                                    state_handler=state_handler)
-    return "Success!"
+    response_json = json.dumps(dict(response_not_required=True))
+    return response_json
```

Comment:
@vabs22 is this still the right way to send responses to the outgoing webhooks system?